clm_data/README.md

```diff
@@ -1,21 +1,107 @@
 
 ## Overview
-This repository contains the `surfdata_0.9x1.25_16pfts__CMIP6_simyr2000_c170616.nc` file used in the Community Land Model (CLM) for historical climate modeling. The dataset is tailored for simulations that emphasize natural vegetation dynamics without the inclusion of cultivated crops (CFTs).
+This artifcat contains scripts that process several Community Land Model (CLM) data files and maps vegetation properties based on Plant Functional Types (PFTs).
+Below is a detailed description of each file and its purpose.
 
-## File Description
+<!-- This repository contains the `surfdata_0.9x1.25_16pfts__CMIP6_simyr2000_c170616.nc` file used in the Community Land Model (CLM) for historical climate modeling. The dataset is tailored for simulations that emphasize natural vegetation dynamics without the inclusion of cultivated crops (CFTs). -->
+## Usage
+To recreate the artifact:
+1. Create a python virtual environment
+2. Activate the new virtual env
+3. In the same terminal run `pip install -r requirements.txt`
+4. In the same terminal run `julia --project=. create_artifact.jl`
+
+### Requirements
+- Python >=3.8
+
+## Input Files
+
+### 1. `surfdata_0.9x1.25_16pfts__CMIP6_simyr2000_c170616.nc`
 The netCDF file includes comprehensive environmental data with a focus on vegetation represented through 16 different Plant Functional Types (PFTs). These PFTs play a crucial role in modeling biophysical processes and ecosystem functions within CLM simulations.
 
-### Key Variables Related to PFTs
-- **`PCT_NATVEG`**: This variable gives the percentage of natural vegetation cover across the land units, essential for assessing non-agricultural land cover.
-- **`PCT_NAT_PFT`**: Indicates the percentage composition of each PFT (16) within natural vegetation land units, providing a granular look at vegetation distribution.
+- **Key Variables Related to PFTs**
+    - `PCT_NATVEG`: This variable gives the percentage of natural vegetation cover across the land units, essential for assessing non-agricultural land cover.
+    - `PCT_NAT_PFT`: Indicates the percentage composition of each PFT (16) within natural vegetation land units, providing a granular look at vegetation distribution.
 
 These PFT variables are pivotal for simulating how natural ecosystems respond to historical climatic conditions and can be used to project changes in vegetation patterns due to climatic shifts.
 
-### Data Download
-The dataset can be downloaded from the official input data repository for the Community Climate System Model:
+- **Data Download** The dataset can be downloaded from the official input data repository for the Community Climate System Model:
 [CCSM Input Data Repository](https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/lnd/clm2/surfdata_map/)
 
+### 2. `pft-physiology.c110225.nc`
+Contains physiological parameters for each of the 21 PFTs in a previous version of CLM, when vcmax25 did not depend on nitrogen content.
```

Comment:
The previous section mentions 16 PFTs, this one 21. Which one is correct?

Reply:
Both are correct. These two different versions of PFT maps had different numbers of PFTs.

Reply:
Maybe, add the name of variables and refer to the PFTs classifications.